stubs/resources/views/components/nav-dropdown.blade.php

```diff
@@ -0,0 +1,9 @@
+<li {{ $attributes->merge(['class' => 'dropdown']) }} x-data="{ open: false }" @keyup.escape.window="open = false" @click.away="open = false" @close.stop="open = false">
```

Comment:
What if the dropdown is the only item in the navigation for example when used as a nav menu in mobile widths?

Some possible options:

- have a nested option that toggles the root element as either `<nav>` or `<li>`, will need to provide an option for the `<nav>` label. 
- use a generic root element and expect that when using the component it is nested appropriately.